### PR TITLE
Cleaned up descriptions

### PR DESCRIPTION
**Pull request by aggarwav**

### CODE REVIEW

File: risk_cd.md [hunk 1]
```diff
@@ -14,11 +14,19 @@ lastupdated: "2017-08-01"
 
 # Integrating Deployment Risk analytics with Continuous Delivery
 
-You can instrument pipelines for {{site.data.keyword.contdelivery_full}} to use {{site.data.keyword.DRA_short}}' Deployment Risk analysis capabilities. For more information about Continuous Delivery pipelines, see [the official documentation](../ContinuousDelivery/pipeline_working.html).
+In this page,
+- All references to pipeline are references to Continuous Delivery pipelines. For more information, see [the official documentation](../ContinuousDelivery/pipeline_working.html).
```
Comment (maxbridges) [inline risk_cd.md]: This will need to be reverted given the CD pipeline naming guidelines.

File: risk_cd.md [hunk 2]
```diff
@@ -14,11 +14,19 @@ lastupdated: "2017-08-01"
...
 
-After you add {{site.data.keyword.DRA_short}} to your toolchain, you can instrument your pipelines to publish build, deployment, and test records to it.
+- Deployment Risk Analysis refers to a capability in the IBM DevOps Insights offering.
```
Comment (maxbridges) [inline risk_cd.md]: The About Deployment Risk topic can define this. This is a task topic--the conceptual product stuff is generally confined to the About pages.

File: risk_cd.md [hunk 3]
```diff
@@ -14,11 +14,19 @@ lastupdated: "2017-08-01"
...
 
-After your pipeline starts publishing data to {{site.data.keyword.DRA_short}}, define risk policies. You can enforce risk policies by adding gate jobs to your pipeline. For more information about defining risk policies, see [Creating policies and rules](./risk_policies.html#policies_and_rules).
+
+You can instrument pipeline jobs to leverage Deployment Risk analysis capabilitiey.  Here are the high level steps:
+
+- Add {{site.data.keyword.DRA_short}} to your toolchain.  TODO TODO: Can we add a screenshot of this or link to another documentation?
```
Comment (maxbridges) [inline risk_cd.md]: This is explained in the Getting Started page. In the task topics, it's assumed that the reader has already added the service to the toolchain by following the instructions there. The Bluemix Docs model is not [Every Page Is Page One](http://everypageispageone.com/)--the module needs to be considered as a whole.

Comment (maxbridges) [inline risk_cd.md]: I'l pick this up tomorrow, but the About Deployment Risk topic already has a high-level flow for using DRA. That page can be updated as necessary and linked from both here and the Jenkins wiki stuff.

**Comment by maxbridges** [comment]
Made a few comments, but haven't finished. Will pick this up in the morning.

I think that Bluemix Docs' new information architecture can make describing certain processes rather difficult without either fragmenting the subject matter or bogging readers down with details. 

I wonder if a more conversational blog post or article could shoulder some of the burden here.

File: risk_cd.md [hunk 4]
```diff
@@ -79,13 +87,39 @@ Make sure that application names and environments match where appropriate. For e
 ## Publishing test data to DevOps Insights
 {: #configure_pipeline_jobs}
 
-You can publish test data from all job types. {{site.data.keyword.DRA_short}} uses the published data to generate reports and enforce policies at gates.
+You can publish test data from all job types. {{site.data.keyword.DRA_short}} uses the published data to generate reports and enforce risk policies.
+
+There are two options for publishing test results.
```
Comment (maxbridges) [inline risk_cd.md]: @aggarwav When would I use the CLI, and when would I use the an Advanced Tester test job? There are two options, but I'm not sure when to use each. 

Also, are the formats specified in the table in "Publishing test data from Advanced Tester jobs" true for the CLI, too? If so, that information can be moved under the H2 for this section.

Comment (aggarwav) [inline risk_cd.md]: @maxbridges Yes, same formats are supported.

Advanced Tester is really for backward compatibility.

File: risk_cd.md (hunk 2 above)
Comment (aggarwav) [inline risk_cd.md]: @maxbridges cool

**Comment by maxbridges** [comment]
Changing target branch to make working on this a bit easier for me.

**Pull request merged**